src/Toolkit/TheRug.idr

```diff
@@ -8,7 +8,7 @@
 |||
```

Comment:
Generally, I am happy with changes to `TheRug`. As background `TheRug` was inspired by Idris2's own core, which was also specialised. I suppose for the programs I make with `TheRug` the specialisation isn't required.